spec/serializers/category_list_serializer_extension.rb

```diff
@@ -13,8 +12,15 @@
     filter_tag.update(category_ids: [category.id, private_category.id].join("|"))
   end
 
-  it "only includes categories allowed per permissions" do
-    json = CategoryListSerializer.new(CategoryList.new, scope: Guardian.new(user), root: false).as_json
-    expect(json[:categories].pluck(:id)).to eq([category.id])
+  context "only includes categories allowed per permissions" do
+    it "for regular users" do
+      json = CategoryListSerializer.new(CategoryList.new, scope: Guardian.new(user), root: false).as_json
+      expect(json[:categories].pluck(:id)).to eq([category.id])
+    end
+
+    it "for anons" do
```

Comment:
just for clarification, what does this test given that this is not a request with a GTF passed to it?

Reply:
True, these cases are being tested by `spec/requests/categories_controller_spec.rb`